internal/interpreters/python_requires.go

```diff
@@ -53,7 +54,23 @@ func (p *PyProjectPythonRequires) readPythonVersionFile() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	return strings.TrimSpace(string(data)), nil
+
+	parts := strings.Split(string(data), ",")
+	var adapted []string
+
+	for _, part := range parts {
+		part = strings.TrimSpace(part)
+		if part == "" {
+			continue
+		}
+		adaptedPart, err := adaptPythonRequires(part)
+		if err != nil {
+			return "", err
+		}
+		adapted = append(adapted, adaptedPart)
+	}
+
+	return strings.Join(adapted, ","), nil
```

Comment:
It would be really nice to see tests around the various cases that we expect and how they should be transformed.

Reply:
Extended the testsuite with most cases in `TestGetPythonRequiresPythonVersionAdapt`